### ISSUE TITLE
Issue using objc_property

### ISSUE THREAD

**Issue by cculianu** [pybee/rubicon-objc#98]
I have an class roughly like this:

```
class MyObj(NSObject):
    foo = objc_property()

    @objc_method
    def init(self):
        self = ObjCInstance(send_super(self, 'init'))
        self.foo = True
        return self

```

Later on whenever I want to write to self.foo, I get errors such as:
```

app_packages/rubicon/objc/runtime.py", line 1382, in __getattr__
    type(self).__module__, type(self).__qualname__, self.objc_class.name, name)
AttributeError: rubicon.objc.runtime.ObjCInstance __NSCFNumber has no attribute isEqualTo_
(lldb) 
```

I basically can't really overwrite any properties.  I suspect there is some bug in runtime.py.  Any thoughts?

I'm on rubicon_objc-0.2.10-py3.5

Please help!

**Comment by cculianu** [comment]
I figured out why this is happening.  IsEqualTo: is only available on macOS.  I'm using rubicon on iOS. An alternate method needs to be used rather than 'IsEqualTo:'...

Rubicon is broken for iOS until this is fixed.  Perhaps I can issue a pull request to fix this?

Or care to fix it?

**Comment by freakboy3742** [comment]
Please turn down the emotional hyperbole. Rubicon for iOS isn't "broken". It's "broken for your specific use case". I can point at many places where it works fine.

And pull requests are always welcome :-)

**Comment by cculianu** [comment]
Sorry, I didn't mean to offend you.  I wasn't being emotional, it's just the way I talk, man. :)

Rubicon is great.  I am happy it exists and it does actually work like magic most of the time.  It's just this 1 little bug.

I'm going to submit a PR now, but I am not sure you'll be entirely happy with it.  It's worth a shot, though.  Basically isEqualTo: is missing from NSObject on iOS.  Here's the Apple developer page for this method.. notice how platforms is *only* "macOS".

https://developer.apple.com/documentation/objectivec/nsobject/1393823-isequalto?language=objc

So my workaround is to basically not call it, and instead do the assignment using a .autorelease()/.retain() pair.  It works.. but maybe you will find it breaks semantics.  We'll see.

Coming shortly...

**Comment by riazrizvi** [comment]
If I may, I don't see why it is desirable to inject a MacOS function to this iOS interface, it would make the API confusing. Why not just use the available IsEqual function? Personally I don't see any need for a change.

**Comment by dgelessus** [comment]
Fixed in #99.
